src/component/Profile.js

```diff
@@ -0,0 +1,61 @@
+import { useSelector, useDispatch } from 'react-redux';
+import './styles/profile.css';
+
+const Profile = () => {
+  const dispatch = useDispatch();
+  const rocketsList = useSelector((state) => state.rockets.rockets);
+  const reservedRockets = rocketsList.filter((rocket) => rocket.reserved === true);
+
+  const { missions } = useSelector((state) => state.mission);
+  const reservedMission = missions.filter((mission) => mission.reserved === true);
+
+  return (
+    <div className="reserved-rockets-con">
+      <div className="reserved-table">
+        <h3>My Rockets</h3>
+        <div className="list-con">
+          {reservedRockets.length === 0 ? (
+            <span>Not reserved rockets!</span>
+          ) : (
+            <ul>
+              {reservedRockets.map((rocket) => (
+                <li className="reserved-list" key={rocket.id}>{rocket.rocket_name}</li>
+              ))}
+            </ul>
+          )}
+        </div>
+      </div>
+      <div className="reserved-mission">
+        <h3>My Missions</h3>
+        <div className="mission-list-container">
+          {reservedMission.length === 0 ? (
+            <p>No Missions Yet!</p>
+          ) : (
+            <ul className="mission-lists">
+              {reservedMission.map((mission) => (
+                <li className="mission-list" key={mission.mission_id}>
+                  {mission.mission_name}
+                  <span className="leave-profile-btn">
+                    {mission.reserved && (
+                      <button
+                        type="button"
+                        className="leave-btn"
+                        onClick={() => {
+                          dispatch(leaveMission(mission.mission_id));
+                        }}
+                      >
+                        Leave Mission
+                      </button>
```

Comment:
The `leave mission` button could have been left here by mistake since it is not part of the actual design. Also clicking on it leads to an error in the website. Kindly consider removing it. to match the design 

### Your design

![Screenshot from 2023-06-10 08-11-56](https://github.com/emmanuelsteven/spaceMission/assets/65233860/497ff7b4-47cd-48b6-a78c-7d918937f712)



### Actual design 

![Screenshot from 2023-06-10 08-11-47](https://github.com/emmanuelsteven/spaceMission/assets/65233860/072e6b32-79ff-4d8e-992f-c17ec5265728)